internal/pkg/agent/operation/tests/scripts/serviceable-1.0-darwin-x86_64/main.go

```diff
@@ -136,6 +136,7 @@ func clientFromNet(port int, impl client.StateInterface, actions ...client.Actio
 		ServerName:   connInfo.ServerName,
 		Certificates: []tls.Certificate{cert},
 		RootCAs:      caCertPool,
+		MinVersion:   tls.VersionTLS12,
```

Comment:
Since we are controlling everything could we use `VersionTLS13`, that should be available on endpoint side too?

Reply:
We could, this change really just came in to fix lint. Would prefer to not make that change in the PR, just made the change for lint.